modules/core/src/wallet.ts

```diff
@@ -929,6 +929,31 @@ Wallet.prototype.signTransaction = function (params, callback) {
     throw error;
   }
 
+  const externalSignerUrl = this.bitgo._externalSignerUrl;
+  if (externalSignerUrl) {
+    const transactionHex = params.transactionHex;
+    const unspents = params.unspents;
+    const keychain = params.keychain;
+    if (!_.isString(params.transactionHex)) {
+      throw new Error('expecting the transaction hex as a string');
+    }
+    return Bluebird.resolve(fetch(externalSignerUrl, {
```

Comment:
Why use `fetch` instead of `superagent`, which is what we use for making HTTP requests elsewhere in the package?